clang/test/Preprocessor/embed_parsing_errors.c

```diff
@@ -94,6 +94,9 @@ char buffer[] = {
 #embed "embed_parsing_errors.c" prefix() // OK: tokens within parens are optional
 #embed "embed_parsing_errors.c" prefix)
 // expected-error@-1 {{expected '('}}
+#embed "embed_parsing_errors.c" prefix()) // expected-error {{expected identifier}}
```

Comment:
We should also make sure to handle unbalanced open braces as well: `prefix(()` and `suffix({)` etc.

Reply:
Oh my eyes glazed over the existing coverage, sorry!